React Front End

jagernet-ops opened the pull request:
# React Front End Pull Request
## Assignee: MJS
### Info-for-Herrmann:
Inside of this PR entails all details of the React development. Assign issues accordingly!

jagernet-ops committed:
instantiated a new NextJS project

jagernet-ops committed:
deleted boiler plate code and added placeholder text

jagernet-ops commented:
> Is this ready for review?

You can merge the current changes. The web app has placeholder values currently.

jagernet-ops committed:
removed arbitrary styling

jagernet-ops committed:
Merge branch 'python-42:main' into react-development

jagernet-ops committed:
Delete assets/css directory

CSS will be stored within the fast-fish web app directory in the future

jherrmann06 commented on fast-fish/public/manifest.json:
```diff
@@ -0,0 +1,25 @@
+{
+  "short_name": "React App",
```
Change to "Fast Fish"

jherrmann06 commented on fast-fish/public/manifest.json:
```diff
@@ -0,0 +1,25 @@
+{
+  "short_name": "React App",
+  "name": "Create React App Sample",
```
Change to a longer name including "Fast Fish"

jherrmann06 reviewed (changes requested):
Mostly looks good, just a tiny change

jagernet-ops committed:
deleted redundant src folder and updated boilerplate manifest

jagernet-ops commented:
> Mostly looks good, just a tiny change

I have implemented the change to the manifest file's name, I deleted the old src folder which I glossed over(oops), and I changed the logo images within the public folder to be fishes of different sizes!

jagernet-ops committed:
Delete src directory

jagernet-ops committed:
added title component, deck images, turn images, and implemented hover animations

jagernet-ops committed:
finished clipping and attaching card icons

jagernet-ops committed:
implemented poker-esque UI elements, and implemented input boxes

jagernet-ops committed:
added bot name input, and refactored CenterBoard component code

jagernet-ops committed:
fill bothand prototype function

jherrmann06 reviewed (approved):
Already did a more informal review in person so I just skimmed this. Looks good
